internal/snmp/translate.go

```diff
@@ -111,22 +111,30 @@ type MibEntry struct {
 }
 
 func TrapLookup(oid string) (e MibEntry, err error) {
-	var node gosmi.SmiNode
-	node, err = gosmi.GetNodeByOID(types.OidMustFromString(oid))
+	var givenOid types.Oid
+	if givenOid, err = types.OidFromString(oid); err != nil {
+		return e, fmt.Errorf("Could not convert OID %s: %w", oid, err)
```

Comment:
Error strings should start with a lowercase letter. ;-) Same for the other instances.